README.md

```diff
@@ -1,4 +1,37 @@
 # rc-spanishIdValidator [Work in progress] 
 
-This is and AngularJS directive which provides validation for common Spanish official ID numbers like DNI or NIF.
+This is and AngularJS library which provides validation directives for common Spanish official ID numbers like DNI or NIF.
 
+###Usage and examples
+
+The usage of the validation directives is very easy.
+Each input you want to validate must contain the **ng-model** and the directive, **rc-dni** in this example. Now the directive will set invalid the input if it is not correct.
```

Comment:
I have a question about the snapshot. Can we use other directives in the example ? Because our directive don't show if its wrong or not by itself, so we need validation summary or ng-messages.

We can't add the link to the tutorial because we don't have tutorial yet.